src/ForgotPasswordController.js

```diff
@@ -52,7 +54,10 @@ function (Okta, FormController, Enums, FormType, ValidationUtil, Util, ContactSu
       }
     },
     getTemplateData: function () {
-      return this.settings.pick('helpSupportNumber');
+      return _.extend(
```

Comment:
no need for `_.extend` here, we could just return the object with the data we need
```
return {
  helpSupportNumber: this.settings.get('helpSupportNumber'),
  hideBackToSignInForReset: this.settings.get('features.hideBackToSignInForReset')
}
```